components/application/index.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+import { Container, AppBar, SideBar } from '../layout'
+import RepositorySelect from './repository-select'
+import ActivitySelect from './activity-select'
+
+const Application = ({ children, ...restProps }) => (
+  <>
+    <Container {...restProps}>
+      <AppBar>
+        <RepositorySelect />
+      </AppBar>
+
+      <SideBar>
+        <h2>Navigate your data</h2>
```

Comment:
Is it going to stay here?

Reply:
Should be for screen reader only. I did not have this utility class for this moment.

Reply:
https://github.com/oacore/design/issues/61